### ISSUE TITLE
Log levels do not follow the predefined values from the standard library

### ISSUE THREAD

**Issue by ss-remygreinhofer** [datafolklabs/cement#533]
# Cement Issue Reporting and Feature Requests

**System Information**
```bash
$ cement --version
Cement Framework 3.0.0
Python 3.7.0
Platform Darwin-17.7.0-x86_64-i386-64bit
```

**Steps to Reproduce (Bugs Only)**

I am trying to convert a numerical value to a log level. The use case behind it is to be able to specify multiple `-v` to increase the verbosity of the application (i.e. `-vvv`). 

However the levels defined the [logging module](https://github.com/datafolklabs/cement/blob/master/cement/ext/ext_logging.py#L95) do not exactly match the levels defined in the [stdlib](https://docs.python.org/3/library/logging.html#logging-levels). `FATAL` does not exist in the stdlib, and `NOTSET` and `CRITICAL` are missing in the cement logging extension.

It would be convenient and logical to make those lists identical.

**Example**

_Put a breakpoint in the command you use to test._

```python
>>> verbose_level
50
>>> import logging
>>> logging.getLevelName(verbose_level)
'CRITICAL'
>>> self.app.log.set_level(level_name)
>>> self.app.log.get_level()
'INFO'
```
If I use another level it works as expected:
```python
>>> verbose_level
40
>>> import logging
>>> logging.getLevelName(verbose_level)
'ERROR'
>>> self.app.log.set_level(level_name)
>>> self.app.log.get_level()
'ERROR'
```

**Comment by derks** [comment]
@ss-remygreinhofer the Cement Logging Extension is slightly simplified wrapper around logging...  in this case, I think the issue is that we only used/supported `FATAL` and did not include direct support for `CRITICAL`.  If you try to set a level that isn't supported it just defaults to `INFO`:

- https://github.com/datafolklabs/cement/blob/master/cement/ext/ext_logging.py#L133


Does it work for you to simply use `FATAL` instead of `CRITICAL`?

**Comment by ss-remygreinhofer** [comment]
I can work with this, but I think it would be more clear if the framework was using the same levels as the stdlib.

**Comment by derks** [comment]
This has been merged to `main` and will be released in `stable/3.0.10`. 

- https://docs.builtoncement.com/release-information/deprecations#3.0.10-1



Verified using the above example code:

```
def command1(self):
        """Example sub-command."""

        import logging

        level = 20
        print(logging.getLevelName(level))
        self.app.log.set_level('CRITICAL')
        print(self.app.log.get_level())

        self.app.log.critical('This is a critical message')
        
        # now deprecated
        self.app.log.fatal('This is also a critical message')
```

Result:

```
|> cement-py312 <| myapp # myapp command1
/src/cement/utils/version.py:102: DeprecationWarning: datetime.datetime.utcfromtimestamp() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.fromtimestamp(timestamp, datetime.UTC).
  timestamp = datetime.datetime.utcfromtimestamp(int(timestamp))
/src/cement/core/foundation.py:762: DeprecationWarning: App.Meta.framework_logging will be changed or removed in Cement v3.2.0. See: https://docs.builtoncement.com/release-information/deprecations#3.0.8-2
  deprecate('3.0.8-2')
INFO
CRITICAL
CRITICAL: This is a critical message
/src/cement/ext/ext_logging.py:363: DeprecationWarning: The FATAL logging facility is deprecated in favor of CRITICAL, and will be removed in future versions of Cement.. See: https://docs.builtoncement.com/release-information/deprecations#3.0.10-1
  deprecate('3.0.10-1')
CRITICAL: This is also a critical message
```
